CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 [Full Changelog](https://github.com/folio-org/ui-finance/compare/v3.0.0...v3.0.1)
 
 * Users having difficulty finding fiscal years as they don't know what to search. Refs UIF-335.
+* Display "Do not restrict _____" ledger toggles in ledger view. Refs UIF-348.
```

Comment:
`Do not restrict encumbrance/expenditures` instead of `_______ ` looks much better 😉

Reply:
Yeah, it is)